kerastuner/utils.py

```diff
@@ -11,28 +11,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""KerasTuner utilities."""
```

Comment:
As a side note: the official name is "Keras Tuner" in 2 words.

Reply:
Oh ok, thanks changed!